apps/server/src/app.ts

```diff
@@ -69,6 +72,23 @@ app.use(function (err: HttpError, req: Request, res: Response) {
   res.render("error");
 });
 
+// Define routes for registering as a customer or a rider
+app.post('/register/customer', async (req, res) => {
+  await registerCustomer(req, res);
+});
+
+app.post('/register/rider', async (req, res) => {
+  await registerRider(req, res);
+});
+
+// Define route for retrieving rider details
+app.get('/riders', async (req, res) => {
+  await getRiderDetails(req, res);
+});
+app.get('/rider:id', async (req, res) => {
+  await getRiderById(req, res);
+});
+
```

Comment:
There's a routes folder for you to do all this. There's no need to implement this in the app.ts file. Remove this please and update the routes folder accordingly.